### DISCUSSION TITLE
Backup/Export scripts

### DISCUSSION THREAD

**Discussion by XxR3K1xX** [amidaware/tacticalrmm#1767] [Q&A] [answered]
Hello!,
Anyone knows if it is possible to backup or exports scripts for importing them in another tactical instance?
Only scripts, I don´t want to restore the entire Tactical RMM configuration.
Thanks!

**Comment by dinger1986** [comment]
they are stored in the database, do they contain secret information? You could add them to the community scripts?

What is the use purpose?

**Reply by XxR3K1xX** [reply]
Hi!
Thank you so much for the reply, I´ll try to backup scripts from database 🫶
As a sysadmin in my organization I have a lot of custom scripts, most of them are for the following actions:

- (Win & Linx) Downloading third party apps from official repositories and install or uninstall them silently.
- (Win & Linx) Gather relevant information from the computers (MAC addresses, installed software, connected peripherals...)
- (Win) Auto upgrading winget apps
- (Linx) Check permissions
- etc

**Reply by dinger1986** [reply]
All sounds useful for community scripts if you were happy sharing them :)

**Reply by XxR3K1xX** [reply]
Of course! I'll post them when I have some time.

**Comment by NiceGuyIT** [comment, marked as answer]
I created a [taskfile][1] to [export and import scripts][2] from Tactical. The problem is that the import treats them the same as community scripts and thus read-only.

I haven't touched the script in several releases and it may take a lot of TLC to work. If you only want the [sql to export][3] from the database, it's there.

[1]: https://github.com/go-task/task
[2]: https://github.com/NiceGuyIT/taskfiles/tree/main/trmm
[3]: https://github.com/NiceGuyIT/taskfiles/blob/main/trmm/My-Scripts.yaml#L257

**Reply by XxR3K1xX** [reply]
Ok, thank you so much, I´ll try it 😉
